src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Patterns.cs

```diff
@@ -106,7 +106,7 @@ private BoundExpression CompareWithConstant(BoundExpression input, BoundExpressi
                 );
         }
 
-        BoundExpression MakeDeclarationPattern(SyntaxNode syntax, BoundExpression input, LocalSymbol target, bool requiresNullTest)
+        BoundExpression MakeDeclarationPattern(SyntaxNode syntax, BoundExpression input, BoundExpression target, bool requiresNullTest)
```

Comment:
That is not possible, as there is no way to evaluate an expression where a type parameter is not bound to a ground type. Some of the others are possible.

---
In reply to: [77389609](https://github.com/dotnet/roslyn/pull/13535#discussion_r77389609) [](ancestors = 77389609)